services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerStateModel.java

```diff
@@ -73,6 +81,8 @@ public VeniceControllerStateModel(
     this.realTimeTopicSwitcher = realTimeTopicSwitcher;
     this.accessController = accessController;
     this.helixAdminClient = helixAdminClient;
+    this.workerService = Executors
+        .newSingleThreadExecutor(new DaemonThreadFactory("Venice-Controller-State-Model-Worker-" + this.clusterName));
```

Comment:
Can you get `LogContext` from `multiClusterConfigs` and pass it to `DaemonThreadFactory`?  Also can you shorten the thread name. It's too long 😄